### PR TITLE
VT-5721 3dCart: Unable to update User Key and Token

### PR DESCRIPTION
**Pull request by mrassudishkin**
[VT-5721]

Added new config with URL to v2 version 3DCart.
Also updated pagination. 3DCart returns error if we try receive page where empty data. So I added condition that we should stop to receive data if count of items in page less than our pageSize.

[VT-5721]: https://agileharbor.atlassian.net/browse/VT-5721?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dk-skuvault — APPROVED** [review]

File: src/ThreeDCartAccess/RestApi/Models/Configuration/IRestThreeDCartConfig.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+namespace ThreeDCartAccess.RestApi.Models.Configuration
+{
+	public interface IRestThreeDCartConfig
```
Comment (andrey-kozlov-skuvault) [inline src/ThreeDCartAccess/RestApi/Models/Configuration/IRestThreeDCartConfig.cs:3]: Why do you need this interface at all? Is `RestThreeDCartConfigBase` not enough?

Comment (mrassudishkin) [reply src/ThreeDCartAccess/RestApi/Models/Configuration/IRestThreeDCartConfig.cs:3]: Don't want use abstract class RestThreeDCartConfigBase for it. I think that interfaces are better solution for it. Third-side projects like v1 should see interfaces, not abstract classes.

Comment (andrey-kozlov-skuvault) [reply src/ThreeDCartAccess/RestApi/Models/Configuration/IRestThreeDCartConfig.cs:3]: @mrassudishkin Still it is a bit strange for me. It is not a service with logic, it is just an settings/config object with almost no logic and behavior. V1 could use specfic classes if v1 wants to instantiate it. Also, in abstract class, virtual methods could be mocked, if needed.
So I still do not see a need for such an interface for config object without logic. If we create interface for this, it should/could be replaced with other implementation. Is there a reason for v1 or inside Access project to substitute this with other service, or mock? I thought 2 existing descendants are good enough for all our cases, but maybe I miss something.
So in general I'm not sure if we need to introduce interface for something without behavior (validation of input does not look like a logic we may want to substitute in tests or in v1).

Comment (mrassudishkin) [reply src/ThreeDCartAccess/RestApi/Models/Configuration/IRestThreeDCartConfig.cs:3]: Ok. First I did just abstract class but I was sure that everyone will want interface here :)) Will fix.

Comment (mrassudishkin) [reply src/ThreeDCartAccess/RestApi/Models/Configuration/IRestThreeDCartConfig.cs]: @andrey-kozlov-skuvault done

**Review by andrey-kozlov-skuvault — APPROVED** [review]

**Pull request merged**